src/siminterface/simulator.py

```diff
@@ -48,6 +52,8 @@ def init(self, seed):
         self.env = simpy.Environment()
         self.params = SimulatorParams(self.network, self.ing_nodes, self.sfc_list, self.sf_list, self.config,
                                       self.metrics)
+        self.params.metrics.reset_metrics()
+        self.params.metrics.reset_run_metrics()
```

Comment:
You can/should drop this line as run metrics are reset within the `reset_metrics` call too: https://github.com/RealVNF/coordination-simulation/blob/master/src/coordsim/metrics/metrics.py#L53